monetio/obs/aeronet.py

```diff
@@ -146,16 +146,21 @@ def add_data(
         interp_to_aod_values=interp_to_aod_values,
     )
 
-    requested_parallel = n_procs > 1 or n_procs == -1
-    if has_joblib and requested_parallel:
-        # Split up by day
+    requested_parallel = n_procs > 1 or n_procs <= -1
```

Comment:
```suggestion
    requested_parallel = n_procs != 1
```
Probably this is fine.